appservice/src/createAppService/createAppService.ts

```diff
@@ -75,7 +97,30 @@ export async function createAppService(
     if (showCreatingNode) {
         showCreatingNode(wizardContext.newSiteName);
     }
+    if (!advancedCreation) {
+        const basicPlanSku: SkuDescription = { name: 'B1', tier: 'Basic', size: 'B1', family: 'B', capacity: 1 };
+        const freePlanSku: SkuDescription = { name: 'F1', tier: 'Free', size: 'F1', family: 'F', capacity: 1 };
+        wizardContext.newResourceGroupName = `appsvc_rg_${wizardContext.newSiteOS}_${wizardContext.location.name}`;
+        wizardContext.newPlanName = `appsvc_asp_${wizardContext.newSiteOS}_${wizardContext.location.name}`;
+        // Free tier is only available for Windows
+        wizardContext.newPlanSku = wizardContext.newSiteOS === WebsiteOS.windows ? freePlanSku : basicPlanSku;
+    }
     wizardContext = await wizard.execute(actionContext, outputChannel);
 
     return wizardContext.site;
 }
+
+async function setWizardContextDefaults(wizardContext: IAppServiceWizardContext): Promise<void> {
+    await workspace.findFiles('package.json').then((files: Uri[]) => {
+        if (files.length > 0) {
```

Comment:
Is 'files' always defined?

Reply:
Yeah, if findFIles doesn't return any files, then it will return an empty array,